bindinfo/bind_serial_test.go

```diff
@@ -78,13 +187,32 @@ func TestBindingSymbolList(t *testing.T) {
 	require.Equal(t, "t:ia", tk.Session().GetSessionVars().StmtCtx.IndexNames[0])
 	require.True(t, tk.MustUseIndex("select a, b from t where a = 3 limit 1, 100", "ia(a)"))
 
+	tk.MustExec("prepare stmt1 from 'select a, b from t where a = 3 limit 1, 100';")
```

Comment:
Please add one more separate test for this feature like `TestPrepareCacheWithBinding`.